worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorldEdit.java

```diff
@@ -135,20 +134,10 @@ private void load(FMLLoadCompleteEvent event) {
         if (FMLLoader.getDist() == Dist.CLIENT) {
             // we want to setup platform before we hit the main menu
             // but this event is async -- so we must delay until the first game loop:
-            Minecraft.getInstance().addScheduledTask(this::setupPlatform);
+            //Minecraft.getInstance().enqueue(this::setupPlatform);
```

Comment:
If you're going to drop this for sure, I would remove the surrounding method as well. If it's working without it, might as well remove it all.